autograder/api/submission/removesubmission.py

```diff
@@ -0,0 +1,32 @@
+import autograder.api.common
+import autograder.api.config
+
+API_ENDPOINT = 'submission/remove/submission'
+API_PARAMS = [
+    autograder.api.config.PARAM_COURSE_ID,
+    autograder.api.config.PARAM_USER_EMAIL,
+    autograder.api.config.PARAM_USER_PASS,
+    autograder.api.config.PARAM_ASSIGNMENT_ID,
+
+    autograder.api.config.PARAM_TARGET_EMAIL,
```

Comment:
According to your backend API contract, this should be PARAM_TARGET_EMAIL_OR_SELF.